_pytest/terminal.py

```diff
@@ -44,9 +44,9 @@ def pytest_addoption(parser):
                      help="traceback print mode (auto/long/short/line/native/no).")
     group._addoption('--show-capture',
                      action="store", dest="showcapture",
-                     choices=['no', 'stdout', 'stderr', 'both'], default='both',
-                     help="Controls how captured stdout/stderr is shown on failed tests. "
-                          "Default is 'both'.")
+                     choices=['no', 'stdout', 'stderr', 'log', 'all'], default='all',
```

Comment:
Good thinking, just realized that we definitely should have used `all` instead of `both` from the beginning. 😉